cli/cmd/install-cni-plugin.go

```diff
@@ -0,0 +1,106 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"text/template"
+
+	installcniplugin "github.com/linkerd/linkerd2/cli/cmd/install-cni-plugin"
+	"github.com/spf13/cobra"
+)
+
+type installCNIPluginConfig struct {
+	Namespace           string
+	CNIPluginImage      string
+	LogLevel            string
+	InboundPort         uint
+	OutboundPort        uint
+	IgnoreInboundPorts  string
+	IgnoreOutboundPorts string
+	ProxyUID            int64
+}
+
+func newCmdInstallCNIPlugin() *cobra.Command {
+	options := newCNIPluginOptions()
+
+	cmd := &cobra.Command{
+		Use:   "install-cni [flags]",
+		Short: "Output Kubernetes configs to install the Linkerd CNI Plugin",
+		Long: `Output Kubernetes configs to install the Linkerd CNI Plugin.",
+This command installs a Daemonset into the Linkerd control plane. The Daemonset
```

Comment:
Let's add an additional newline before this paragraph, and change `Daemonset` to `DaemonSet` throughout.